app/src/main/java/org/mozilla/fenix/BrowserDirection.kt

```diff
@@ -21,7 +21,7 @@ enum class BrowserDirection(@IdRes val fragmentId: Int) {
     FromSyncedTabs(R.id.syncedTabsFragment),
     FromBookmarks(R.id.bookmarkFragment),
     FromHistory(R.id.historyFragment),
-    FromExceptions(R.id.exceptionsFragment),
+    FromTrackingProtectionExceptions(R.id.trackingProtectionExceptionsFragment),
```

Comment:
a lot of the churn here is I tried to make it less confusing by renaming all of the "exceptions" before to what they are -> tracking protection exceptions